rasa/shared/constants.py

```diff
@@ -84,3 +84,4 @@
 DEFAULT_CONVERSATION_TEST_PATH = "tests"
 
 DIAGNOSTIC_DATA = "diagnostic_data"
+DEFAULT_RANDOM_SEED = 29306
```

Comment:
why is this in `rasa.shared.constants`? Isn't `rasa.constants` enough? Or do we expect to share this constant with downstream dependencies such as Rasa X?

Reply:
No strong opinions/preferences - can certainly move it to `rasa.constants`.

Reply:
cool!